### PR TITLE
Added approvalPrompt to properties file

### PR DESCRIPTION
**Pull request by ffarrabal**

### CODE REVIEW

**Comment by luisromangz** [comment]
:+1:

File: src/main/java/com/emergya/spring/security/oauth/google/GoogleAuthCodeResourceDetails.java [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+ * See https://developers.google.com/youtube/v3/guides/authentication for more
+ * info.
  *
  * @author lroman
  */
 public class GoogleAuthCodeResourceDetails extends AuthorizationCodeResourceDetails {
 
-    private String approvalPrompt = "auto";
+    @Value("${google.approvalPrompt}")
+    private String approvalPrompt;
```
Comment (luisromangz) [inline src/main/java/com/emergya/spring/security/oauth/google/GoogleAuthCodeResourceDetails.java]: If you use the property in GoogleOAuth2SecurityConfiguration you don't need to set it here I believe.

**Comment by ffarrabal** [comment]
You're right. Fixed.

**Comment by luisromangz** [comment]
:ok_hand:

**Pull request merged**